Saule/Serialization/ErrorRenderers/DbEntityValidationExceptionRenderer.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Entity.Validation;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Saule.Serialization.ErrorRenderers
+{
+    internal class DbEntityValidationExceptionRenderer : ExceptionRenderer

```

Comment:
As said above, I think this belongs in people's projects, not Saule. We don't want to depend on entity framework.